recipes/haploview/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "haploview" %}
+{% set version = "4.2" %}
+
+about:
```

Comment:
can you please move this down, so that it follows the ordering of other recipes? Thanks!